Source/AudioVisualizer.cpp

```
@@ -176,6 +205,66 @@ void AudioVisualizer::setAudioData(const float* leftChannel, const float* rightC
     currentSample_ = (currentSample_ + samplesToCopy) % displayBufferSize_;
 }
 
+void AudioVisualizer::setPeakLevels(float leftPeak, float rightPeak, float leftRMS, float rightRMS) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'setPeakLevels' of similar type ('float') are easily swapped by mistake

(bugprone-easily-swappable-parameters)